### PR TITLE
Fix/alliance deck in normal archonte

### PR DESCRIPTION
**Pull request by Osarek**

### CODE REVIEW

File: client/Components/Decks/DeckList.jsx [hunk 1]
```diff
@@ -185,6 +185,14 @@ const DeckList = ({ deckFilter, onDeckSelected, standaloneDecks = false }) => {
                     value: data.filters[k].filterVal
                 }));
 
+                //filter comming from elsewhere than the table change
+                if (
```
Comment (cryogen) [inline client/Components/Decks/DeckList.jsx:189]: if(deckFilter?.isAlliance)

**Comment by Osarek** [comment]
there is 3 case of calling this method

1/ Decks page : deckFilter?.isAlliance=undefined
--> we don't need to add the filter
2/ Deck selection for Alliance : deckFilter?.isAlliance=true
--> we need to add the filter
3/ Deck selection for Normal: deckFilter?.isAlliance=false
--> we need to add the filter


should i do
if (deckFilter?.isAlliance == false || deckFilter?.isAlliance == true) 
or 
if (typeof deckFilter.isAlliance !== 'undefined' )

**Comment by cryogen** [comment]
> 

I see what you're trying to do.  Just checking if(deckFilter) should be enough in that case

**Comment by Osarek** [comment]
I think that should be what I want
Did not test that for the moment 

if (deckFilter?.isAlliance?)

**Comment by Osarek** [comment]
> > 
> 
> I see what you're trying to do. Just checking if(deckFilter) should be enough in that case

You're right, that's enough to cover the 3 cases.
thanks

**Pull request merged**